### DISCUSSION TITLE
not support zero-width negative lookahead assertion

### DISCUSSION THREAD

**Discussion by zornsoon** [projectdiscovery/nuclei#4349] [Q&A] [answered]
Hello, I found that when I run nuclei, I want to match the content of the returned packet, but it seems that the zero-width negative lookahead assertion is not supported to filter out the information I want to filter.
![image](https://github.com/projectdiscovery/nuclei/assets/61613298/ce256393-5d9f-49db-8869-04db25bedde7)

The running results are as follows:
![image](https://github.com/projectdiscovery/nuclei/assets/61613298/227e0e12-5271-44dd-a3a8-0a9d1c2cf6ea)

**Comment by geeknik** [comment, marked as answer]
The Go programming language does not support zero-width negative lookahead assertions in its regular expressions. This is due to technical reasons, specifically because it conflicts with the O(n)-time guarantees of the library. More info [here](https://groups.google.com/g/golang-nuts/c/7qgSDWPIh_E) and [here](https://swtch.com/~rsc/regexp/regexp3.html). 🤙🏻
